notification/twilio/validation.go

```diff
@@ -25,7 +25,14 @@ func validateRequest(req *http.Request) error {
 
```

Comment:
would it be worth adding additional validation returned if the user has `AlternateAuthToken` set and not `AuthToken`?

something like `"missing primary X-Twilio-Signature. only alternate token found."?

Reply:
It shouldn't be necessary; if Twilio is enabled, AuthToken must always be set; Config validation enforces it.

However, that did point out that I forgot to validate the AlternateAuthToken, just pushed a fix for that.